server/service/src/sync/translations/clinician_store_join.rs

```diff
@@ -71,18 +71,27 @@ impl SyncTranslation for ClinicianStoreJoinTranslation {
         let ClinicianStoreJoinRow {
             id,
             store_id,
-            clinician_id,
+            clinician_id: clinician_link_join,
         } = ClinicianStoreJoinRowRepository::new(connection)
             .find_one_by_id_option(&changelog.record_id)?
             .ok_or(anyhow::Error::msg(format!(
                 "Clinician row ({}) not found",
                 changelog.record_id
             )))?;
 
+        let clinician_link_row = ClinicianLinkRowRepository::new(connection)
```

Comment:
Used `ClinicianLinkRowRepository` since didn't want to add a lot of code for query just to use `ClinicianStoreJoinRepository`...

Reply:
Yes the clinician_id of the clinician store join is used nowhere, is it? why is that?

Reply:
Yeah, not used anywhere... There is a sub query in clinicians to get store clinicians...